packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-basic-sign-in/main/aspnet/integrationsteps.md

```diff
@@ -2,7 +2,7 @@
 
 After you complete the steps in the [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/aspnet/main/) use case and the Widget loads successfully, the next step is for the user to sign in. There is no additional code that you need to add to your app for this step. The user enters their credentials and clicks the **Next** or **Sign In** button.
 
-<div class="common-image-format">
```

Comment:
Hrm, I have checked this out, and it is behaving like I expected — at widths of less than 768px, it goes to three-quarter width. I chose 768px because that seems to be the mobile breakpoint being used elsewhere, kind of on the assumption that people are unlikely to view it at this width on a desktop computer ;-)

I wonder should I apply the 350px max-width to half width images all the time, and not just when the viewport is more than 768px.

In fact, I've made the change, Let me know if you think this works better.

Reply:
Odd - I can't recreate the issue now. Possibly it was my browser acting oddly. Anyway it works fine now.